### PR TITLE
Apply alphabetic sorting

### PR DESCRIPTION
**Pull request by LyubomirT**
This update introduces alphabetical sorting for items in the ListView component. The sorting logic is applied when setting the items. Hope this is helpful!

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -59,6 +59,11 @@ static void SetupCallbacks(ListView& albumViewRef, ListView& songViewRef)
             songVec.push_back({ cols, file.path() });
         }
 
+        // Sort the items alphabetically
+        std::sort(songVec.begin(), songVec.end(), [](const auto& a, const auto& b) {
```
Comment (BonuxPink) [inline src/main.cpp:63]: Nit: Move '{' to next line.

Comment (BonuxPink) [inline src/main.cpp:63]: It would be nice if you made std::sort into std::ranges::sort, i try to be consistent with that.

**Review by BonuxPink — CHANGES_REQUESTED** [review]

**Comment by LyubomirT** [comment]
@BonuxPink Just made the necessary changes, please let me know if you need anything else.

**Comment by BonuxPink** [comment]
All good

**Pull request merged**